astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx

```diff
@@ -9,38 +9,38 @@ tags: webauthn, passkeys, passwordless
+* Passkeys help resolve significant security problems related to data breaches, phishing, and attacks against 2FA methods. Phishing attacks are usually disguised emails or messages luring users to click on malicious links with the intent of stealing sensitive data like login credentials. With WebAuthn, attackers can't gain access to your data using login credentials stolen from your device since passkeys use cryptographic keys to authenticate users and the private key never leaves the device nor is accessible to the user.
 * It's easier to use since users have to manage fewer passwords.
-* It eliminates the need for servers to store passwords. With passwords, if a server happens to be compromised, the user might be at risk if the password wasn’t hashed properly. With WebAuthn, if the server is compromised, the server only stores the public key, which requires the private key (stored in the user's device) to complete the authentication.
+* It eliminates the need for servers to store passwords. With passwords, if a server happens to be compromised, the user might be at risk if the password wasn’t hashed properly. With passkeys, if the server is compromised, the server only stores the public key, which requires the private key (stored in the user's device) to complete the authentication.
 * It's easier for attackers to steal a user's password than to steal their device. Even if an attacker does steal a device, they still need to bypass the device's security (for example, biometrics or facial recognition system).
-* WebAuthn is easy to implement because it consists of [browser APIs that can be easily used by developers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#examples).
+* Passkeys are easy to implement because it consists of [browser APIs that can be easily used by developers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#examples).
 * It offers a smooth authentication experience for users.
 
-## How WebAuthn Works
+## How Passkeys Work
 
-WebAuthn consists of three components: the authenticator (the user's device), the web browser, and the server (RP). In essence, the user initiates a login via the browser, the browser then prompts the user to authenticate using their device (via fingerprint or facial recognition, for example), and then if successful, the user is logged into the web application.
+Passkey authentication consists of three components: the authenticator (the user's device), the web browser, and the server (RP). In essence, the user initiates a login via the browser, the browser then prompts the user to authenticate using their device (via fingerprint or facial recognition, for example), and then if successful, the user is logged into the web application.
```

Comment:
Same with this one. The swap doesn't work here. 


Passkey = credential but this describes WebAuthn as a whole

Reply:
Reverted